src/compas_timber/connections/solver.py

```diff
@@ -1,5 +1,6 @@
 import itertools
 import math
+import stat
```

Comment:
imported but not used, could that be?

Reply:
ooh VScode keeps adding stupid imports >_<